config/settings.yml

```diff
@@ -54,3 +54,12 @@ skylight:
   enable: false
   authentication: please_change_me
 render_json_errors: false
+
+personas:
+  admin:
+    email_address: "becomingateacher+admin-integration-tests@digital.education.gov.uk"
+    first_name: "Support agent"
+    last_name: "Colin"
+
+basic_auth:
+  enabled: false
```

Comment:
Need to add `basic_auth: enabled: true` to `settings/qa.yml` so that we can test this out on QA.

Reply:
Also need to add `authentication: mode: persona` to `settings/qa.yml`.